torch/nn/functional.py

```diff
@@ -3788,10 +3787,17 @@ def multi_head_attention_forward(query,                           # type: Tensor
           the embedding dimension.
         - value: :math:`(S, N, E)` where S is the source sequence length, N is the batch size, E is
           the embedding dimension.
-        - key_padding_mask: :math:`(N, S)`, ByteTensor, where N is the batch size, S is the source sequence length.
+        - key_padding_mask: :math:`(N, S)` where N is the batch size, S is the source sequence length.
+          If a ByteTensor is provided, the positions with the value of non-zero will be ignored while the position
```

Comment:
"the positions with the value of non-zero" can be written more concisely as "the non-zero positions".

Reply:
Also applies to other parts in the documentation.